yarn.lock

```diff
@@ -4558,7 +4570,7 @@ class-utils@^0.3.5:
     isobject "^3.0.0"
     static-extend "^0.1.1"
 
-classnames@^2.2.5:
+classnames@^2.2.5, classnames@^2.2.6:
```

Comment:
mind checking why we have two classnames dependencies in the project?

Reply:
how do you check such a thing?

Reply:
yarn.lock
It will show you who depends on what.

Reply:
yes I know this is what we see here but I added "classnames": "^2.2.6" - why it keeps the 2.2.5?

Reply:
You didn't look :)
If you look at yarn.lock you will see that react-draggable is using it which is used by storybook.
We're good.